Gemfile.lock

```diff
@@ -106,7 +106,20 @@ GEM
       railties (>= 4.1.0)
       responders
       warden (~> 1.2.3)
+    devise-jwt (0.9.0)
```

Comment:
- [ ] It would be better if the signup and login functionality works as expected. I'm unable to authenticate and use the application